ovn_k8s/modes/overlay.py

```diff
@@ -304,3 +312,124 @@ def add_endpoint(self, event):
             return
 
         self._update_vip(service_data, ips)
+
+    def sync_pods(self, pods, namespace):
+        expected_logical_ports = []
+        if pods and 'items' in pods:
+            for pod in pods['items']:
+                pod_name = pod['metadata']['name']
+                logical_port = "%s_%s" % (namespace, pod_name)
+                expected_logical_ports.append(logical_port)
+
+        try:
+            existing_logical_ports = ovn_nbctl(
+                                "--data=bare", "--no-heading",
+                                "--columns=name", "find",
+                                "logical_switch_port",
+                                "external_ids:namespace=" + namespace).split()
+        except Exception as e:
+            vlog.err("sync_pods: find failed %s" % (str(e)))
```

Comment:
is there a reason for not raising or returning here? I don't see how the sync process can successfully complete